src/features/musl.ts

```diff
@@ -3,9 +3,14 @@ import * as core from '@actions/core'
 import * as tc from '@actions/tool-cache'
 import {exec} from '../utils'
 import {join} from 'path'
+import {homedir} from 'os'
+import {promises as fs} from 'fs'
 
-const MUSL_NAME = 'x86_64-linux-musl-native'
```

Comment:
We only supported musl on x86_64 only before?

Reply:
We only support static images on ["Linux x64 operating system"](https://github.com/oracle/graal/blob/6dab549194b85252f88bda4ee825762d8b02c687/docs/reference-manual/native-image/guides/build-static-and-mostly-static-executable.md#prerequisites-and-preparation).